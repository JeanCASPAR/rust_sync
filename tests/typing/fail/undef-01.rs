use rust_sync::sync;

sync! {
    node oui() = (hello)
    where
        bye : int = 3;
}

fn main() {}
