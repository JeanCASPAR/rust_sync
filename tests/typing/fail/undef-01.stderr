error: Variable `hello` is undefined.
 --> tests/typing/fail/undef-01.rs:4:19
  |
4 |     node oui() = (hello)
  |                   ^^^^^
